chia/rpc/wallet_rpc_api.py

```diff
@@ -1338,13 +1338,18 @@ async def create_new_dl(self, request) -> Dict:
                     self.service.wallet_state_manager.main_wallet,
                 )
 
-        dl_tx, std_tx, launcher_id = await dl_wallet.generate_new_reporter(
-            bytes32.from_hexstr(request["root"]), fee=request.get("fee", uint64(0))
-        )
+        try:
+            dl_tx, std_tx, launcher_id = await dl_wallet.generate_new_reporter(
+                bytes32.from_hexstr(request["root"]), fee=request.get("fee", uint64(0))
+            )
+        except ValueError as e:
+            log.error(f"Error while generating new reporter {e}")
+            return {"success": False, "error": str(e)}
```

Comment:
Doesn't the framework handle this such that just a bare `raise` would be fine after logging?  Or maybe the framework takes care of the logging as well?  The log should have the traceback.